scripts/demo.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# For now, we use this script to start our demo on rederer
+# by changing the ENTRYPOINT and CMD at the dockerfile to this.
+
+# If no parameter is passed, use https://demo.bytebase.com:80 as host by default
+ONLINE_DEMO_HOST=$1
```

Comment:
I would prefer to use a single argument for host:port. Because the script doesn't support named argument, so it's less error prone to just have a single argument.

On the other hand, if we have 2 arguments, ./demo.sh 80 https://example.com would be incorrect.